Terminal.Gui/Views/FileDialog.cs

```diff
@@ -485,11 +486,14 @@ private void New ()
 
 
 		/// <inheritdoc/>
-		public override bool ProcessHotKey (KeyEvent keyEvent)
+		public override bool OnHotKey (KeyEventArgs keyEvent)
 		{
-			if (this.NavigateIf (keyEvent, Key.CtrlMask | Key.F, this.tbFind)) {
+#if BROKE_IN_2927
+			// BUGBUG: Ctrl-F is forward in a TextField. 
```

Comment:
Now that we have keybinding support, I think we can get rid of some of the less standard shortcuts like:

```csharp
AddKeyBinding (Key.F | Key.CtrlMask, Command.Right);
AddKeyBinding (Key.E | Key.CtrlMask, Command.RightEnd);
AddKeyBinding (Key.B | Key.CtrlMask, Command.Left);
```

These are very common key combinations that users might bind specific functionality in a parent view.  If that functionality works until you tab into a `TextField` (at which point the sub view swallows the key rather than it bubbling up) then that would be confusing for both users and programmers.

In the case of file dialog specifically, I want it to mirror desktop dialogs where pressing Ctrl+F anywhere moves the cursor to the search control.

Reply:
I really never liked `Ctrl-F` in the `TextField`. It's used a lot in macOS, but it can changed as @tznind  said, to `Alt-F`.

Reply:
I actually think the proper behavior for moving to any textfield field is to modify label to support hotkeys and use tabstop so that if a label gets a command, it focuses on the next view in the tab order. 

In file dialog you have no label before the search box, which is good for a screen Real Estate perspective, but bad for a discoverability perspective. We could make it so that the label associated with a text field can be zero sized and it'll still work.  I tried Implementing this, but I couldn't get it to work and realized I was just being distracted so I backed out.  I plan on opening an issue to describe a proposed design.

All of the navigation complexity in file dialog is complex and confusing. I'm curious why you didn't just depend on the tab stop tab index functionality.  

I discovered something else doing this, which is there are times where defining a custom command would be useful. Do you recall why you chose not to support Custom commands?

Based on what I've done so far, I'm pretty confident that all of the hot/cold key stuff can be removed and there was still will be flexibility for doing super custom stuff. This makes me happy because it's a massive simplifier.

Reply:
I have opened a discussion where we can talk in more detail about the navigation and other ux choices in FileDialog https://github.com/gui-cs/Terminal.Gui/discussions/3020